api/python/t4/packages.py

```diff
@@ -443,16 +443,20 @@ def set_dir(self, lkey, path):
         if url.scheme == 'file':
             src_path = pathlib.Path(parse_file_url(url))
             files = src_path.rglob('*')
+            has_added_file = False
```

Comment:
I'm not sure this logic is correct, I think that `set_dirs` should except empty directories but raise exceptions on directories that don't exist?